src/Microsoft.ML.OnnxTransform/OnnxTransform.cs

```diff
@@ -189,16 +226,32 @@ public IRowToRowMapper GetRowToRowMapper(Schema inputSchema)
             return MakeDataTransform(new EmptyDataView(_host, inputSchema));
         }
 
+        private static int[] AdjustDimensions(OnnxShape shape)
+        {
+            if (shape.Count > 0)
```

Comment:
>shape.Count > 0 [](start = 16, length = 15)

is shape.Count == 0 a valid case or an error? right now it's being treated as a valid case. If it's valid please comment about what kind of models would have it.

Reply:
Added a comment -- let me know if it makes sense.

---
In reply to: [234307353](https://github.com/dotnet/machinelearning/pull/1586#discussion_r234307353) [](ancestors = 234307353)